src/cli/cli.cpp

```diff
@@ -5018,6 +5018,129 @@ template <> otError Interpreter::Process<Cmd("meshdiag")>(Arg aArgs[])
         SuccessOrExit(error = otMeshDiagDiscoverTopology(GetInstancePtr(), &config, HandleMeshDiagDiscoverDone, this));
         error = OT_ERROR_PENDING;
     }
+    /**
+     * @cli meshdiag childtable
+     * @code
+     * meshdiag childtable 0x6400
+     * rloc16:0x6402 ext-addr:8e6f4d323bbed1fe ver:4
+     *     timeout:120 age:36 supvn:129 q-msg:0
+     *     rx-on:yes type:ftd full-net:yes
+     *     rss - ave:-20 last:-20 margin:80
+     *     err-rate - frame:11.51% msg:0.76%
+     *     conn-time:00:11:07
+     *     csl - sync:no period:0 timeout:0 channel:0
+     * rloc16:0x6403 ext-addr:ee24e64ecf8c079a ver:4
+     *     timeout:120 age:19 supvn:129 q-msg:0
+     *     rx-on:no type:mtd full-net:no
+     *     rss - ave:-20 last:-20  margin:80
+     *     err-rate - frame:0.73% msg:0.00%
+     *     conn-time:01:08:53
+     *     csl - sync:no period:0 timeout:0 channel:0
+     * Done
+     * @endcode
+     * @par
+     * Start a query for child table of a router with a given RLOC16.
+     * Output lists all child entries. Information per child:
+     * - RLOC16
+     * - Extended MAC address
+     * - Thread Version
+     * - Timeout (in seconds)
+     * - Age (seconds since last heard)
+     * - Supervision interval (in seconds)
+     * - Number of queued messages (in case child is sleepy)
+     * - Device Mode
+     * - RSS (average and last)
+     * - Error rates: frame tx (at MAC layer), IPv6 message tx (above MAC)
+     * - Connection time (seconds since link establishment {dd}d.{hh}:{mm}:{ss} format)
```

Comment:
Surround the example format in ` so it's code font on the website

`{dd}d.{hh}:{mm}:{ss}`